### DISCUSSION TITLE
Torrent labels into lists

### DISCUSSION THREAD

**Discussion by codedesperate** [Novik/ruTorrent#2626] [Q&A] [answered]
Hi all, 
I used to use https://github.com/Novik/ruTorrent/wiki/PluginTaddLabel, but I think this broke down when updating to 4.x.x. It also hasn't been updated in a long time. 

I can however see, that rutorrent is natively setting some torrent labels as lists. I use the following syntax "Test1/Test2", where "/" should make it into a list. 

This is what it looks like in my rutorrent:
![billede](https://github.com/Novik/ruTorrent/assets/29740063/35f51c34-3809-4f06-a10b-a6f29fabf718)

As you can see, Sonarr, Radarr, Seeding gets put in a list, whereas Readarr, Other and Lidarr does not. 
It seems there need to be at least two labels in the list, before rutorrent makes the list structure. 

Is it possible to make the list structure even with just one label, so that a label "Test1/Test2" would have a list, so Test1 appears as a title of Test2? Currently to make a list, both "Test1/Test2" and "Test1/Test3" are required.

**Comment by TrimmingFool** [comment, marked as answer]
Have you tried `Settings > Format > Show empty path labels`?

**Reply by codedesperate** [reply]
No, and I just tried and did exactly what I wanted. Thank you for letting me now :D
